### PR TITLE
Adds support for bifrost deployments and DRYing up the code.

### PR DESCRIPTION
**Pull request by SealedSaint**
Also updates README and node version.

### CODE REVIEW

File: example.env [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
-PATH_TO_SHIPPING=/Users/YOURUSERNAME/PATHTOREPO
 PATH_TO_INVENTORY=/Users/YOURUSERNAME/PATHTOREPO
 PATH_TO_JSREPORTS=/Users/YOURUSERNAME/PATHTOREPO
-PATH_TO_ANALYTICS=/Users/YOURUSERNAME/PATHTOREPO
-GITHUB_USERNAME=
-GITHUB_PASSWORD=
+PATH_TO_SERVER=/Users/YOURUSERNAME/PATHTOREPO
+PATH_TO_SHIPPING=/Users/YOURUSERNAME/PATHTOREPO
+
 SHELL_IS_FISH=FALSE
```
Comment (tom-con) [inline example.env]: this can also be removed

File: jakelib/deploy.jake [hunk 2]
```diff
@@ -1,138 +1,105 @@
...
+  const vars = { cluster_name, app_name };
+  const cmds = [
+    `cd ${path}`,
+    ...apps[app_name].cmds,
+  ];
+  return replacer(cmds.join(' && '), vars);
+};
+
+const normalAppDeployCommands = [
+  'docker build -t {{cluster_name}}-{{app_name}} . --no-cache',
```
Comment (tom-con) [inline jakelib/deploy.jake]: now these vars accessible inside the function... why not do normal interpolation and get rid of the replacer function?

Comment (SealedSaint) [reply jakelib/deploy.jake]: ooo, is this the replacer function? I thought it was some special jake thing. I'll look into it.

Comment (tom-con) [reply jakelib/deploy.jake]: yeah this is the handwound interpolation

Comment (SealedSaint) [reply jakelib/deploy.jake]: I'm gonna merge this in and look at replacing the replacer in a different PR.

Comment (tom-con) [reply jakelib/deploy.jake]: 🆗 🙆‍♂ 👌 🔉 💯 🏷 🥇

**Pull request merged**